src/atoms/GrayBox/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import Layout from 'atoms/Layout';
+import StyledWrapper from 'atoms/StyledWrapper';
+import Col from 'atoms/Layout/Col';
+import colors from 'atoms/Color/colors.scss';
+
+import styles from './gray-box.module.scss';
+
+const GrayBox = ({
+  cols,
+  variant,
+  colorDash,
+  children,
+  verticalPadding,
+  leftOffset
+}) => {
+  const classes = cx(
+    styles['gray-box'],
+    styles[`gray-box-lg-${cols}`],
+    variant && styles[variant],
+    colorDash && colors[`color-dash-${colorDash}`],
+    leftOffset && styles[`offset-${leftOffset}`]
+  );
+
+  const styleOverrides = ({ breakpoints }) => {
+    let str = '';
+
+    if (verticalPadding) {
+      str += `
+        padding-top: ${verticalPadding};
+        padding-bottom: ${verticalPadding};
+      `;
+    }
+
+    if (leftOffset) {
+      str += `
+      @media ${breakpoints.mediumUp} {
+        margin-left: calc(100% / 12 * ${leftOffset}) !important;
+        max-width: calc(100% - (100% / 12 * ${cols - leftOffset}) !important;
```

Comment:
Discussed offline.

Reply:
If you run the calculation, you’re actually just going to get a percentage. The only reason you use `calc()` is to calculate the arithmetic between two different units of measure in the browser. Otherwise, you do the math before you print to css

ie `calc(100% - 15px)` <-- ok vs `calc(100% - (100% / 12 * 15))` <-- not this